src/Tools/Source/RunTests/ProcessRunner.cs

```diff
@@ -7,196 +7,196 @@
 using System.Collections.ObjectModel;
 using System.Diagnostics;
 using System.IO;
+using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace RunTests
+namespace RunTests;
```

Comment:
I'm not opposed to using file level namespaces as you've done in this PR. However making this change while 
refactoring this code makes the change **significantly** harder to read. We should undo this and have a separate PR for the style only change.

Reply:
Disabling white space in the PR review significantly helps with this. It's not as hacky as it used to be (where you edited query parameters) so seems like it may be better supported now.